examples/GLUE/configs-roberta-base/stsb.yaml

```diff
@@ -0,0 +1,31 @@
+glue_task: 'stsb'
```

Comment:
how necessary is it really to have one marlin config per glue task? can we do just one config and specify glue_task param by CLI?

Reply:
It is good to have separate configs, as it will be a nightmare to remember the differences and provide them through command line. Another option will be to harcode the configs in source code but I felt config will be cleaner. Also it will help us easily onboard a  similar but non GLUE task.

Reply:
I didn't realize there was different param changes other than glue_task, if that's the case - I agree we should just do different configs.